tests/unit/ds-macro-test.js

```diff
@@ -37,3 +37,32 @@ test('DS macro', function(assert) {
 
   assert.deepEqual(relationships, ['user', 'car']);
 });
+
+class ES6Model extends DS.Model {
+  @attr firstName;
+  @attr({ defaultTo: 'blue' }) lastName;
+  @hasMany user;
+  @belongsTo car;
+}
+
+test('DS macro (ES6)', function(assert) {
```

Comment:
You should probably define the class inside the body of the test itself, see the `@action` decorator test for an example